app/Traits/GatewayMetadataIngestionTrait.php

```diff
@@ -77,27 +82,42 @@ public function getLocalDatasetsForFederatedTeam(GatewayMetadataIngestionService
         ])->get())->keyBy('pid');
     }
 
-    public function deleteLocalDatasetsNotInRemoteCatalogue(Collection $localItems, Collection $remoteItems): void
+    public function deleteLocalDatasetsNotInRemoteCatalogue(Collection $localItems, Collection $remoteItems): bool
     {
+        $this->log('info', 'testing REMOTE collection for LOCAL deletions');
```

Comment:
Just checking, the use of 'testing' here is to mean "checking", and this it to be kept, it's not debug logging that should be removed?

Reply:
No need to change it if so, just clarifying for my understanding of intent